rabbitmq/check.py

```diff
@@ -397,6 +428,15 @@ def alert(self, base_url, max_detailed, size, object_type, custom_tags):
 
         self.event(event)
 
+    def _limit_vhosts(self, instance):
```

Comment:
See above, I think we already have this info where needed

Reply:
vhosts is populated here: https://github.com/DataDog/integrations-core/pull/910/files/9a6ae0d83cca1f8a04e91c72f5f59acea5fae55d#diff-7b88d57462b0edc9f04cbdba719e5e5eL183

By this function: https://github.com/DataDog/integrations-core/pull/910/files/9a6ae0d83cca1f8a04e91c72f5f59acea5fae55d#diff-7b88d57462b0edc9f04cbdba719e5e5eR162

The function grabs the info about every vhost if it's not limited in the yaml instance.